api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -123,11 +130,17 @@ private boolean expiredByCost(final DbUser user, final double currentCost) {
     return currentCost > getUserFreeTierDollarLimit(user);
   }
 
+  private Optional<Instant> getUserFreeCreditExpirationTime(final DbUser user) {
```

Comment:
We shouldn't be calling this if the registration time is null, but I wanted to make the nullability contract explicit